api_specifications/bail-api-0.1.0.yaml

```
@@ -0,0 +1,264 @@
+    BailSearchResponse:
+      type: object
+      properties:
+        ResponseInfo:
+          $ref: 'https://raw.githubusercontent.com/egovernments/egov-services/master/docs/common-contract/v1/contract.json#/components/schemas/ResponseInfo'
+        bails:
+          type: array
+          items:
+            $ref: '#/components/schemas/Bail'
+        pagination:
+          $ref: 'https://raw.githubusercontent.com/egovernments/DIGIT-Specs/common-contract-update/Common%20Services/common-contract.yaml#/components/schemas/Pagination'
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)